p/themes/Swage/swage.css

```diff
@@ -1266,6 +1251,4 @@ button.as-link {
 
 #slider label {
 	min-height: initial;
-}
-
-/*# sourceMappingURL=swage.css.map */
+}
```

Comment:
Does your editor takes advantage of our `.editorconfig` ? In which case, that would be automatically fixed when saving the file.

Reply:
That was my fault -.-
I use VS Code with the extension https://marketplace.visualstudio.com/items?itemName=yutent.scss-to-css that creates the CSS from SCSS. Unfortunately it cuts this line :/